src/generators/index.ts

```diff
@@ -1,6 +1,7 @@
 // Copyright (c) Jupyter Development Team.
 // Distributed under the terms of the Modified BSD License.
 
-export * from './markdowndocgenerator';
-export * from './latexgenerator';
-export * from './notebookgenerator';
+// Note: keep in alphabetical order...
```

Comment:
Thinking toward a potential future in which there are many (?) ToC generators. In which case, having in alphabetical order allows for faster scanning when looking for, in this case, a particularly export. This may be me being slightly pedantic, but my tendency is to prefer imposing order over disorder.